address/book.go

```diff
@@ -138,6 +138,10 @@ type BookConfig struct {
 	// Chain points to the chain the address.Book is active on.
 	Chain ChainParams
 
+	// ProofCourierAddr is the address of the proof courier service to use
+	// for proof transfer.
+	ProofCourierAddr ProofCourierAddr
```

Comment:
Should we call this `FallbackProofCourierAddr` since we would only use this for addresses that don't have one set?

Reply:
I think `ProofCourierAddr` is the primary source for a courier addr when generating new addresses (using `Book.NewAddress`). In that sense it will not be a fallback.

Reply:
Also perhaps we may want to add several of such instances in the future? Given that maybe we start to add the nostr option or w/e, but not all clients support that. We should think through what that update process looks like and how clients/wallet would handle such a flow.

Reply:
Added tracking issue for design task: https://github.com/lightninglabs/taproot-assets/issues/455